### PR TITLE
chore(docs): upgrade docs for release 1.3.0

### PR DESCRIPTION
**Pull request by skie1997**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [x] Site / documentation update
- [x] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: docs/assets/examples/en/axis/symlog-axis.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+cover: https://lf9-dp-fe-cms-tos.byteorg.com/obj/bit-cloud/vchart/preview/axis/symlog-axis.png.jpeg
+option: lineChart#axes
+---
+
+# symlog axis
+
+A Symlog axis is a type of axis used for drawing charts that balances between logarithmic and linear axes while retaining the advantages of logarithmic axes and linear axes.
+The Symlog axis is characterized by symmetry, that is, the scale lines of positive and negative values ​​are symmetrically distributed on both sides of the axis, which helps to better display the positive and negative relationship of the data. The tick marks on the Symlog axis are usually distributed on a logarithmic axis, but in the area close to 0, the tick marks are converted to a linear axis distribution to better display the absolute value of the data.
+
+## critical configuration
```
Comment (kkxxkk2019) [inline docs/assets/examples/en/axis/symlog-axis.md:16]: 这个统一成 ## Key option 把

File: docs/assets/examples/en/axis/symlog-axis.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    ]
+  };
+const vchart = new VChart(spec, { dom: CONTAINER_ID });
+vchart.renderAsync();
+
+// Just for the convenience of console debugging, DO NOT COPY!
+window['vchart'] = vchart;
+```
+
+## 相关教程
```
Comment (kkxxkk2019) [inline docs/assets/examples/en/axis/symlog-axis.md:163]: 这里没有转成英文

File: docs/assets/examples/en/axis/symlog-axis.md (hunk 1 above)
Comment (skie1997) [reply docs/assets/examples/en/axis/symlog-axis.md:16]: ok

File: docs/assets/examples/en/axis/symlog-axis.md (hunk 2 above)
Comment (skie1997) [reply docs/assets/examples/en/axis/symlog-axis.md:163]: ok

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**